tools/log4shell/main.go

```diff
@@ -130,6 +130,10 @@ func main() {
 						Name:  "archives",
 						Usage: "Only scan for known vulnerable archives. By default the CLI will scan for class files which are known to be vulnerable which will result in higher signal findings. If you are specifically looking for vulnerable Java archive hashes, use this option.",
 					},
+					&cli.BoolFlag{
+						Name:  "processes",
```

Comment:
I think this description should include the caveat that this only works on linux.  We have a LOT of macos and windows users.

Reply:
and maybe the errors that throw in regards to failing to find proc/fs should mention the OS limitation also